english/makesmoores/makesmoores

```diff
@@ -0,0 +1,3 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.